charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 18.1.0
+version: 18.1.1
```

Comment:
```suggestion
version: 18.1.2
```

Reply:
Bump to 19.0.1 since someone had bump the major first :D